lightningd/jsonrpc.c

```diff
@@ -792,3 +796,107 @@ void setup_jsonrpc(struct lightningd *ld, const char *rpc_filename)
 	/* Technically this is a leak, but there's only one */
 	notleak(io_new_listener(ld, fd, incoming_jcon_connected, ld));
 }
+
+/**
```

Comment:
Since you are copying this from the `wallet/walletrpc.c` file, it is best to also remove the function there and have wallet `withdraw` RPC call into `json_tok_address_scriptpubkey`.

Reply:
This also has the nice advantage that it would also fix #927, since Segwit `bcrt1` addresses will now be understood by `withdraw`